### ISSUE TITLE
Not working on IE ?

### ISSUE THREAD

**Issue by jsgagne** [mattboldt/typed.js#310]
### Description

It's working fine on every browser, but not on IE11. It's blocking my script... Am I doing something wrong ? I asked for help on Stack Overflow and get no answer... Maybe you could help me ?

I'm using typed.js on this website : https://howareyoudoing.today/

I added in my script.js :

```
var typed = new Typed('#typed', {
  stringsElement: '#typed-strings',
  typeSpeed: 75
});

```
And this in my html : 

```
<div id="typed-strings">
  <p class="element" style="display:none">How are you doing today ?</p>
</div>
<h1 id="typed"></h1>
```

Thank's !

**Comment by trumfio** [comment]
I cannot get this to work in IE either...I have pulled the latest version of the typed.js Any ideas.?

**Comment by nowakpiotrek** [comment]
Did you find a solution? I'm experiencing the same issue.

**Comment by trumfio** [comment]
No, I couldn't get it to work, so I ditched altogether and went to [http://macarthur.me/typeit/](http://macarthur.me/typeit/). This one works well.

**Comment by mattboldt** [comment]
Should be fixed on v2.0.6 with https://github.com/mattboldt/typed.js/issues/293
